cmd/oras/internal/option/remote.go

```diff
@@ -77,14 +78,15 @@ func applyPrefix(prefix, description string) (flagPrefix, notePrefix string) {
 // Commonly used for non-unary remote targets.
 func (opts *Remote) ApplyFlagsWithPrefix(fs *pflag.FlagSet, prefix, description string) {
 	var (
-		shortUser     string
-		shortPassword string
-		shortHeader   string
-		flagPrefix    string
-		notePrefix    string
+		shortUser          string
+		shortPassword      string
+		shortIdentityToken string
+		shortHeader        string
+		flagPrefix         string
+		notePrefix         string
 	)
 	if prefix == "" {
-		shortUser, shortPassword = "u", "p"
+		shortUser, shortPassword, shortIdentityToken = "u", "p", "i"
```

Comment:
Should not enable shorthand for `--identity-token`

Reply:
Sure will do it